feat(guide): Recommended naming scheme - Add Enable Rename instructions (#1800)
TRaSH-Guides/Guides: docs/Sonarr/Sonarr-recommended-naming-scheme.md

![Unhide Advanced](images/unhide-advanced.png)

> And as last enable Rename Episodes to reveal the episode renaming options.

Reviewer: ```suggestion
Then, enable Rename Episodes to reveal the episode renaming options.
```

> And as last enable Rename Episodes to reveal the episode renaming options.

Reviewer 2: ```suggestion
Finally, enable `Rename Episodes` to reveal the episode renaming options.
